src/collectors/windows.plugin/metadata.yaml

```diff
@@ -1944,12 +1944,12 @@ modules:
               dimensions:
                 - name: blocked
         - name: MSSQL Resource Locks
-          description: "TODO"
+          description: "SQL Server locks on individual resource types. Locks are held on SQL Server resources, such as rows read or modified during a transaction, to prevent concurrent use of resources by different transactions: https://learn.microsoft.com/en-us/sql/relational-databases/performance-monitor/sql-server-locks-object?view=sql-server-ver17."
           labels:
             - name: mssql_instance
               description: The instance name.
             - name: resource
-              description: "TODO: ???"
+              description: "The MSSQL instance name."
```

Comment:
`resource` is not "MSSQL instance name", right?

Reply:
:facepalm: Of course. I confused the lines. Thanks.